### ISSUE TITLE
Extension `empty history` button broken in 3.18.1.1

### ISSUE THREAD

**Issue by heftig** [Keruspe/GPaste#131]
Clicking the button just causes these messages:
```
(gnome-shell:1704): Gjs-WARNING **: JS ERROR: Error: Too few arguments to method GPaste.empty_history expected 2 got 1
GPasteEmptyHistoryItem<._init/<@/usr/share/gnome-shell/extensions/GPaste@gnome-shell-extensions.gnome.org/emptyHistoryItem.js:39
```

gnome-shell 3.18.1
gpaste 3.18.1.1

**Comment by Keruspe** [comment]
Fixing it, I know what's wrong.

Good timing, I was going to do a release in a couple of hours

**Comment by heftig** [comment]
Is it because the method is async and expects a callback?

**Comment by Keruspe** [comment]
There's actually two problems here: emptyHistory now takes the name of the history as a parameter (already fixed this one locally)
Second problem is empty action seems to be broken, fixing it

**Comment by Keruspe** [comment]
Ok, actually, it's emptying the current history which is broken, fixing it

**Comment by Keruspe** [comment]
Fixed by 07ede508a556915ca6c4caeb9ad741593d6c3d7d and d5da82f02ffc2eaaef398101859b9ec680b5ab89

Will do a 3.18.2 later today, thanks for the report!
